dissect/cstruct/parser.py

```diff
@@ -109,7 +110,7 @@ def _enum(self, tokens: TokenConsumer) -> None:
         values = {}
         for line in d["values"].splitlines():
             for v in line.split(","):
-                key, sep, val = v.partition("=")
+                key, _, val = v.partition("=")
```

Comment:
```suggestion
                key, val = v.split("=", 1)
```
No need for the dummy variable right?

Reply:
`split` has different behaviour, and this suggestion will break if no `=` is present in `v`. Whereas with `partition` you will always get 3 string values returned, albeit with `sep` or `val` being empty strings.

Reply:
Ok, yes you are right.